inquire/src/prompts/path_select/action.rs

```diff
@@ -0,0 +1,73 @@
+use crate::{
+    ui::{Key, KeyModifiers},
+    InnerAction, InputAction,
+};
+
+use super::config::PathSelectConfig;
+
+/// Set of actions for a MultiSelectPrompt.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum PathSelectPromptAction {
+    /// Action on the value text input handler.
+    FilterInput(InputAction),
+    /// Moves the cursor to the option above.
+    MoveUp,
+    /// Moves the cursor to the option below.
+    MoveDown,
+    /// Moves the cursor to the page above.
+    PageUp,
+    /// Moves the cursor to the page below.
+    PageDown,
+    /// Moves the cursor to the start of the list.
+    MoveToStart,
+    /// Moves the cursor to the end of the list.
+    MoveToEnd,
+    /// Toggles the selection of the current option.
+    ToggleCurrentOption,
+    /// Selects all options.
+    SelectAll,
+    /// Deselects all options.
+    ClearSelections,
+    /// Navigates deeper in file tree
+    NavigateDeeper,
+    //// Navigated higher in file tree
```

Comment:
```suggestion
    /// Navigates higher in file tree.
    ///
    /// No-op if on a root directory.
```

wdyt? I know it's pretty obvious but I like the "being very explicit" approach